docs/data/joy/customization/theme-typography/theme-typography.md

```diff
@@ -76,46 +114,10 @@ declare module '@mui/joy/styles' {
 }
 ```
 
-## Usage
-
-There are several ways that you can use the theme typography in your application:
-
-- [Typography](/joy-ui/react-typography/) component: use the `level` prop to change between theme typography levels:
-
-  ```jsx
-  // use the `theme.typography['body-sm']` styles
-  <Typography level="body-sm">Secondary info</Typography>
-  ```
-
-- [CSS Baseline](/joy-ui/react-css-baseline/) component: by default, it applies the `body1` level to the global stylesheet:
-
-  ```jsx
-  <CssBaseline />
-
-  // inherits the `theme.typography['body-md']` styles
-  <p>Hello World</p>
-  ```
-
-- [`sx`](/joy-ui/customization/approaches/#sx-prop) prop: use `typography: $level` to get the specific theme typography level:
-
-  ```jsx
-  // to apply the `theme.typography['body-sm']` styles:
-  <Box sx={{ typography: 'body-sm' }}>Small text</Box>
-  ```
-
-- [`styled`](/joy-ui/customization/approaches/#reusable-component): create a custom component and apply the style from `theme.typography.*` directly:
-
-  ```jsx
-  import { styled } from '@mui/joy/styles';
+### Changing the default font
 
-  const Tag = styled('span')((theme) => ({
-    ...theme.typography['body-sm'],
-    color: 'inherit',
-    borderRadius: theme.vars.radius.xs,
-    boxShadow: theme.vars.shadow.sm,
-    padding: '0.125em 0.375em',
-  }));
-  ```
+Joy UI uses the [Inter](https://rsms.me/inter/) font by default.
+To change it, do this...
```

Comment:
Here's the specific request left as feedback 😬 @siriwatknp will need your help to quickstart this section!